fe/fe-core/src/main/java/com/starrocks/statistic/ExternalFullStatisticsCollectJob.java

```diff
@@ -216,6 +227,7 @@ private void flushInsertStatisticsData(ConnectContext context, boolean force) th
             context.setExecutor(executor);
             context.setQueryId(UUIDUtil.genUUID());
             context.setStartTime();
+            context.getSessionVariable().setQueryTimeoutS(20);
```

Comment:
Why set 20s query timeout

Reply:
we just insert the result to be, don't need too much time.